app/services/ocean_service.rb

```diff
@@ -0,0 +1,66 @@
+class OceanService
+  ITERATIONS = 8
+  SEED_POLYGON_OPTIONS = { sides: 5, center: [500, 500], radius: 100 }
+
+  def call
+    inline = create_polygon
+    outline = Geometry::Polygon.new(*inline.vertices)
+    {
+      islands:  [
+        {
+          points_outline: convert_to_array_points(outline.vertices),
+          points_inline:  convert_to_array_points(inline.vertices)
```

Comment:
maybe we should change the format of this so we can have multiple terrains later:
```
{
  islands: [
    {
      terrains: {
        beach: { 
          points: [...]
        },
        forest: {
          points: [...]
        }
      }
    }
  ]
}
```